wrangler-core/src/main/java/co/cask/wrangler/registry/DirectiveInfo.java

```diff
@@ -111,21 +114,4 @@ public UsageDefinition definition() {
   public Directive instance() throws IllegalAccessException, InstantiationException {
     return (Directive) directive.newInstance();
   }
-
-  public JsonObject toJson() {
-    JsonObject response = new JsonObject();
-    response.addProperty("plugin", name);
-    response.addProperty("usage", usage);
-    response.addProperty("description", description);
-    response.addProperty("class", directive.getCanonicalName());
-    response.add("definition", definition.toJson());
```

Comment:
This did not include `optionalCnt` before. Should it be transient?

Reply:
done